wsi/drm/drmbuffer.cpp

```diff
@@ -78,28 +78,12 @@ DrmBuffer::~DrmBuffer() {
 }
 
 void DrmBuffer::Initialize(const HwcBuffer& bo) {
-  width_ = bo.width_;
-  height_ = bo.height_;
-  for (uint32_t i = 0; i < 4; i++) {
-    pitches_[i] = bo.pitches_[i];
-    offsets_[i] = bo.offsets_[i];
-    gem_handles_[i] = bo.gem_handles_[i];
-  }
-
-  format_ = bo.format_;
-  if (format_ == DRM_FORMAT_NV12_Y_TILED_INTEL || format_ == DRM_FORMAT_NV21)
```

Comment:
You removed these checks ?

Reply:
These checks are for setting format_ which is no longer needed since the shorthand aliases are being removed.

Reply:
We still need this as these are custom formats on Android which we translate to standard DRM enums.

Reply:
Is it just format_ that needs to be preserved?

Reply:
I think u forgot to update the PR ?

Reply:
Hmmm, I thought I did so.  Let me check tomorrow.